### PR TITLE
Stormshield - Improve SNS parser

### PR DESCRIPTION
**Pull request by TonioRyo**

### CODE REVIEW

File: Stormshield/stormshield_network_security/ingest/parser.yml [hunk 1]
```diff
@@ -42,44 +42,197 @@ stages:
           event.kind: "event"
           event.category: ["network"]
           event.type: ["connection"]
+          event.dataset: "{{ kv.result.logtype }}"
           "@timestamp": "{{ parsed_time.date }}"
           observer.serial_number: "{{ kv.result.fw }}"
           event.timezone: "{{ kv.result.tz }}"
           event.risk_score: "{{ kv.result.pri }}"
-          stormshield: "{{ kv.result }}"
+          #stormshield: "{{ kv.result }}"
```
Comment (squioc) [inline Stormshield/stormshield_network_security/ingest/parser.yml:50]: I think we can remove this line
```suggestion
```

**Review by squioc — CHANGES_REQUESTED** [review]
My bad. Just the additional comma to remove in the smart-descriptions

File: Stormshield/stormshield_network_security/ingest/parser.yml [hunk 2]
```diff
@@ -42,44 +42,197 @@ stages:
...
+          event.category: ["process"]
+          stormshield.alarm.id: "{{ kv.result.alarmid }}"
+        filter: '{{ kv.result.logtype == "system" }}'
+
+      - set:
+          event.type: ["info"]
+          event.category: ["configuration"]
+          error.code: "{{ kv.result.error }}"
+          source.ip: "{{ kv.result.address }}"
+          client.ip: "{{ kv.result.address }}"
```
Comment (PierrickV) [inline Stormshield/stormshield_network_security/ingest/parser.yml:126]: it is better to protect IP add fields with the ` | is_ipaddress"` filter :pray:

**Review by squioc — APPROVED** [review]
LGTM

**Pull request merged**